### DISCUSSION TITLE
Satranç turnuvası başarı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#283] [Zeka Sorusu]
10 kişinin katıldığı bir satranç turnuvasında her galibiyet 2 puan, beraberlik 1 puan, mağlubiyet 0 puandır. Her oyuncu kendi dışındaki kişilerle birer maç yapacaktır. 8 puan kazananın başarılı olduğu turnuvada, başarılı olan öğrenci sayısı en az kaç olabilir?

**Comment by Mertcakm** [comment]
2

**Reply by sonsuzus** [reply]
Soruları açıklamalı çözelim.

**Comment by yukseel** [comment]
![image](https://github.com/sonsuzus/sonsuzus/assets/114495004/b8810ee5-96ff-4cb6-9b97-428a45d4cb9b) 5 oyuncu
